Trim introductions, add conclusions (#36)
data-lessons/instructor-training: _episodes/23-introductions.md

> - "Introductions should both include practical information and start building relationships."

Reviewer: falgging that this section should all be updated to reflect new content too,

> Optional question: what did you (the leader) do or not do in your introduction to the session?
{: .discussion}

> ### Learning Objectives For your Introduction

Author: I added some introductory text to explain why this header is here. I'm not sure that makes it less confusing?

> ### Learning Objectives For your Introduction

Reviewer: this makes sense to me now.

After the introduction learners should:

- be able to predict the type of instruction
- know what will be taught
- understand what will be required of them
- believe that they can learn from the workshop

The instructional team should:

- know of who is taking the workshop and what their expectations are

> - have an initial impression of how learners respond to prompts and what will be needed to encourage them

Reviewer 2: This point could be a bit clearer. What do we mean about respond to prompts and what will be needed to encourage them?

Author: Tried to clarify this with "participation prompts" and "encourage them to engage"

> ### Firing Up a Learning Environment

Reviewer 2: Not sure what Firing Up a Learning Environment means. Is this a continuation of the launch metaphor?

Author: Yeeeahhh maybe not such a great one. Would "warming up" be less awkward or should this get dumped entirely?

Author: Yeeeahhh thanks that was weird. Maybe better now?

These objectives can be met in many ways; many of them can be addressed at the same time. As you consider the approach you will take, take a step back to consider the full range of ingredients that combine to make the introduction a complete experience.

> **Your attire.** This is one of those things that "should not matter" but, in reality, clothing has a powerful influence on perceptions of everything from 

Reviewer 2: I appreciate this reimaging of the attire comment.

credibility to kindness. Be comfortable, be intentional, and convey what you want to communicate.

> **The physical environment**. Are you in a small classroom or a large, impersonal lecture hall? How are the seats arranged? Physical features may be beyond your control, but they still contribute to your learner's impressions. How can you help?

Reviewer 2: Probably should make some reference to online environments

Reviewer: +1

Author: ```suggestion
**The physical environment**. Are you in a small classroom or a large, impersonal lecture hall? How are the seats arranged? Physical features may be beyond your control, but they still contribute to your learner's impressions. How can you help? (Note: if you are in an online environment, structural features can also impact your workshop experience! What are those features, and how can you modify or moderate their impact?)
```

> Optional question: what did you (the leader) do or not do in your introduction to the session?
{: .discussion}

### Learning Objectives For your Introduction

It can feel like the Introduction is "just" something you have to get through in order to get started on the "real" workshop content.

> But, chances are that many of the goals you identified in the activity above involved *teaching* information or procedures that

Reviewer: activity above is removed?

you want learners to know, or *learning* things about your audience. This means that everything you have learned about
teaching and learning applies here, too. Short term memory? Check! Cognitive overload? Check! 

> Learning objectives usually omit introductory content -- after all, no one cares what has been learned here once the workshop has
> finished! But if we did include the Introduction, what would those learning objectives look like?

Reviewer: ```suggestion
Learning objectives usually omit introductory content -- after all, these do not typically relate to our long term goals. The content of the introduction may not even be relevant once the workshop has
finished! But if we did include the Introduction, what would those learning objectives look like?
```

After the introduction learners should:

- be able to predict the type of instruction
- know what will be taught
- understand what will be required of them
- believe that they can learn from the workshop

The instructional team should:

> - know of who is taking the workshop and what their expectations are
> - have an initial impression of how learners respond to participation prompts and what will be needed to encourage them to engage

Reviewer: ```suggestion
- know of who is participating in the workshop and what their expectations are
- have an initial impression of how learners respond to participation prompts and what will be needed to encourage them to engage
```

an example might be helpful for the second point here.  I think i get it, but this could be a hard to convey concept in abstract

These objectives can be met in many ways; many of them can be addressed at the same time. As you consider the approach you will take, 
take a step back to consider the big picture. How will learners perceive your classroom environment? A few things to pay attention to 
as you include: 

**Your attire.** This is one of those things that "should not matter" but, in reality, clothing has a powerful influence on perceptions of everything from 
credibility to kindness. Be comfortable, be intentional, and convey what you want to communicate. (Also, check the weather, and make
no assumptions about thermostats in an unfamiliar classroom!)

**The physical environment**. Are you in a small classroom or a large, impersonal lecture hall? How are the seats arranged? Physical features may be beyond your control, but they still contribute to your learner's impressions. How can you help?

**Your use of the few minutes before class.** Is the instructional team present and welcoming, or rushing in frantically to a waiting crowd?

**Your introduction of yourself.** Planning this out can help you be intentional about what to include -- and what not to! A few things you may want to convey include

> * your qualifications

Reviewer: qualifications can evoke credentials and that's not what we mean here.  Maybe we drop this as an item and break down the specific, separate parts that we do want people to focus on? enthusiasm is already there, but maybe "experience with the material, what you do outside of the carpentries (since we're volunteer instructors generally)" etc 

The volunteer point can be important, sometimes learners don't know what the carpentries is and think of the instructor as like a paid external professional trainer.

Author: ```suggestion
* your experience or related areas of interes
```

Head here for [more examples of Icebreakers][icebreakers].

> ### Teaching Your Trajectory: Workshop 101

Reviewer 2: Is this also part of the rocket metaphor? "Teaching Your Trajectory" is a bit confusing.

Clearly establishing expectations at the outset orients learners to participate actively
and helps them use their time productively. Keep in mind that critical information will need to be communicated more than once!
This information should also be present in your workshop documentation and/or collaborative notes.

**Describe the prerequisites** (if any).

**Share the schedule and logistics.** Post lunch and break times, and stick to them! Bathroom & lactation room locations and any other 

> instructions specific to your workshop. Demonstrating a commitment to accessibility at this point will help learners feel more comfortable making 

Reviewer 2: "Bathroom & lactation room locations and any other 
instructions specific to your workshop." is an incomplete sentence. Maybe "Share Bathroom and lactation room locations and any other instructions specific to your workshop. "

additional requests as needed.

**Communicate the workshop structure**, including learning objectives and hands-on approach. If your instructional team has distinct roles in the workshop (e.g. 
"notetaker" vs "roaming helper", be sure to introduce them, too.

> **Teach your expectations** for learners, including:

Reviewer 2: Maybe "Communicate your expectations" instead of teach?

Reviewer: ```suggestion
**Communicate your expectations** for learners, including:
```
+1

Head here for [more examples of Icebreakers][icebreakers].

### Teaching Your Trajectory: Workshop 101
Clearly establishing expectations at the outset orients learners to participate actively
and helps them use their time productively. Keep in mind that critical information will need to be communicated more than once!
This information should also be present in your workshop documentation and/or collaborative notes.

**Describe the prerequisites** (if any).

> **Share the schedule and logistics.** Post lunch and break times, and stick to them! Bathroom & lactation room locations and any other 

Author: ```suggestion
**Share the schedule and logistics.** Post lunch and break times, and stick to them! Share bathroom & lactation room locations and any other 
```

>
> This exercise will take about 15 minutes.
{: .challenge}

## The Art of a Smooth Landing
Your workshop almost certainly had some highs and lows. If you followed our advice about going slowly, it is also likely that you will be pressed for time
at the end! Out of respect for all involved, it is important that you end your workshop on time. Even if you have not met your content goals, 
there are far more valuable ways to spend the last 15-20 minutes of a course than squeezing in one last command or bit of advice.

> ## Brainstorm: Making the Last Moments Count
>
> You have made it to the end of your workshop! Everyone is exhausted and their brains are full. You could cover more content... or you could use the last few
> minutes in another way. 
>
> In the etherpad, write down one thing you could do at the end of a workshop. What is the value of spending time on that thing? 
> If you have time after writing down your idea, read through the others in the etherpad. If you have another idea that has not been written down yet, add it 
> to the list.
>
> This exercise will take about 5 minutes.
> > ## Solution

> > > - Reflect on learning. This can help learners to solidify key concepts they have learned, making them easier to remember. It may also flush out a few last questions.

Author: ```suggestion
> > - Close and save files. Where can those files be found, and how can learners pick up independently where you left off?
> > - Reflect on learning. This can help learners to solidify key concepts they have learned, making them easier to remember. It may also flush out a few last questions.
```

> > - Plan next steps. Does the local community have resources to support continued learning? 
> > Do you have advice for how learners might continue on their own? Even
> > if you have no advice, asking learners to take a moment to discuss their own plans can support them in taking a next step sooner rather than later.
> > - Collect feedback. Minute cards, one-up-one-down, and making time for Carpentries post-assessment surveys will support your continuing development as an 
> > Instructor as well as our continuing development of Carpentries programs. This can also support or complement a reflection activity.

> > > - Celebrate everyone's hard work. Thank your learners for helping each other, for staying motivated and persevering with you! Thank your helpers -- keep a

Author: ```suggestion
> > - Check with the workshop host to see if they have any closing words or instructions they would like to share.
> > - Celebrate everyone's hard work. Thank your learners for helping each other, for staying motivated and persevering with you! Thank your helpers -- keep a
```

instructions specific to your workshop. Demonstrating a commitment to accessibility at this point will help learners feel more comfortable making 
additional requests as needed.

**Communicate the workshop structure**, including learning objectives and hands-on approach. If your instructional team has distinct roles in the workshop (e.g. 
"notetaker" vs "roaming helper", be sure to introduce them, too.

**Teach your expectations** for learners, including:
- how to follow the [Code of Conduct](https://docs.carpentries.org/topic_folders/policies/index_coc.html)
- ways to ask for help
- ways to give feedback to the instructional team

**Collect and share baseline data on learners**. If you are teaching an official Carpentries workshop, you should have received the results of your pre-assessment surveys 
-- these can be helpful in planning your workshop! Even so, you may have additional questions, and there will always be learners who did not fill out the survey. 
Sharing and discussing these data with learners can help to combat imposter syndrome and let them know that they are welcome 'as they are'.

> **Share some advice for success** -- including your confidence that they can do it! If you have a range of skill backgrounds, this is a good opportunity to offer 
> differentiated advice on how to make the most of the experience, e.g. suggesting that intermediate learners build their skills by helping a neighbor or
> considering more advanced questions to be discussed during the breaks.

Reviewer: ```suggestion
**Share some advice for success** -- including your confidence that they can do it! If you have a range of skill backgrounds, this is a good opportunity to offer 
differentiated advice on how to make the most of the experience, e.g. suggesting that intermediate learners build their skills by helping a neighbour or
considering more advanced questions to be discussed during the breaks.
```
this might be the point at which you describe that there will be code-along

> This exercise will take 15-25 minutes depending on whether time is included for feedback or not.
{: .challenge}

> ## The Art of a Smooth Landing

Reviewer 2: I like this heading!

Your workshop almost certainly had some highs and lows. If you followed our advice about going slowly, it is also likely that you will be pressed for time

> at the end! For the health and sanity of all involved, it is important that you end your workshop on time. However, even if you have not met your content goals, 

Reviewer 2: Probably good to avoid use of the term sanity.  Maybe say "for the sake of all involved..." instead.

there are far more valuable ways to spend the last 15-20 minutes of a course than squeezing in one last command or bit of advice.

> ## Brainstorm: Making the Last Moments Count
>
> You have made it to the end of your workshop! Everyone is exhausted and their brains are full. You could cover more content... or you could use the last few
> minutes in another way. 
>
> In the etherpad, write down one thing you could do at the end of a workshop. What is the value of spending time on that thing? 

> > If you have time after writing down your idea, read through the others in the etherpad. If you have another idea that hasn't been written down yet, add it 

Reviewer 2: replace hasn't with has not

> This exercise will take 15-25 minutes depending on whether time is included for feedback or not.
{: .challenge}

## The Art of a Smooth Landing

> Your workshop almost certainly had some highs and lows. If you followed our advice about going slowly, it is also likely that you will be pressed for time
> at the end! For the health and sanity of all involved, it is important that you end your workshop on time. However, even if you have not met your content goals, 

Reviewer: ```suggestion
Your workshop almost certainly had some highs and lows. If you followed our advice about going slowly, it is also likely that you will be pressed for time
at the end! Out of respect for all involved, it is important that you end your workshop on time. Even if you have not met your content goals, 
```

this is a good point to maybe put in a formative assessment to ask why that might be better? (or at least note in the curriculum a link back to cognitive load)

there are far more valuable ways to spend the last 15-20 minutes of a course than squeezing in one last command or bit of advice.

> ## Brainstorm: Making the Last Moments Count
>
> You have made it to the end of your workshop! Everyone is exhausted and their brains are full. You could cover more content... or you could use the last few
> minutes in another way. 
>
> In the etherpad, write down one thing you could do at the end of a workshop. What is the value of spending time on that thing? 

> > If you have time after writing down your idea, read through the others in the etherpad. If you have another idea that hasn't been written down yet, add it 

Reviewer: ```suggestion
> If you have time after writing down your idea, read through the others in the etherpad. If you have another idea that has not been written down yet, add it 
```

> to the list.
>
> This exercise will take about 5 minutes.
> > ## Solution
> > - Reflect on learning. This can help learners to solidify key concepts they have learned, making them easier to remember. It may also flush out a few last questions.

> > > - Plan next steps. Does your community have resources to support continued learning? Do you have advice for how learners might continue on their own? Even
> > > if you have no advice, asking learners to take a moment to discuss their own plans can support them in taking a next step sooner rather than later.

Reviewer: ```suggestion
> > - Plan next steps. Does the local community have resources to support continued learning? 
> > Do you have advice for how learners might continue on their own? Even
> > if you have no advice, asking learners to take a moment to discuss their own plans can support them in taking a next step sooner rather than later.
```

> > - Collect feedback. Minute cards, one-up-one-down, and making time for Carpentries post-assessment surveys will support your continuing development as an 
> > Instructor as well as our continuing development of Carpentries programs. This can also support or complement a reflection activity.
> > - Celebrate everyone's hard work. Thank your learners for helping each other, for staying motivated and persevering with you! Thank your helpers -- keep a
> > list of names handy if you might forget them. Enjoy the applause, and give everyone a moment to bask in praise for a job well done.

> > {: .solution}

Reviewer: also should note leaving time for the host as appropriate

and literally how to close, save things, and access again later-- eg jupyter notebooks, review project structure in rstudio, etc.. this is something that's not in a lot of the lessons, but is very important.

{: .challenge}

> 

Reviewer: ```suggestion
In order to make this idea more concrete, the rest of this workshop will be reflection and wrap up.  You may not spend as much time as we're about to in a workshop than we will today, but the idea is the same. 
```
feel free to reject this if it's too meta, but I *think* this "we practice what we teach" thing is a sort of running theme that can be helpful.

Author: I'm going to leave this out for now, because it won't work quite as well if the episodes are reordered (which is likely to happen next). However, I like the meta-take -- it might be good to refer back to this episode in the reflection section instead. (That will also spare some shame for those situations where time gets away and the reflection gets cut from an IT workshop.)
